Tone mapping setup guidance for 10.7 (#442)
jellyfin/jellyfin-docs: general/administration/hardware-acceleration.md

Stream #0:2 -> #0:1 (ac3 (native) -> aac (native))
```

### Configuring OpenCL accelerated Tone mapping

Currently, Tone mapping with Nvidia NVENC, AMD AMF and Intel VAAPI is through OpenCL image support.

OS/Platform | NVIDIA NVENC | AMD AMF  | Intel VAAPI | AMD VAAPI | Intel QSV | Software
----------- | ------------ | -------- | ----------- | --------- | --------- | --------
Linux       | OK           | OK       | OK          | planned   | planned   | planned
Windows     | OK           | OK       | N/A         | N/A       | planned   | planned
Docker      | OK           | untested | OK          | planned   | planned   | planned

> [!NOTE]
> Make sure the hardware acceleration is well configured before reading this section.

> 1. On Windows: Install the latest Nvidia or AMD driver is fairly enough.

Reviewer: ```suggestion
1. On Windows: Install the latest Nvidia or AMD drivers.
```

Stream #0:2 -> #0:1 (ac3 (native) -> aac (native))
```

### Configuring Tone mapping

Tone mapping with Nvidia NVENC and AMD AMF is through OpenCL image support.

>     > [!NOTE]
>     > Make sure the hardware acceleration is well configured before reading this section.

Reviewer 2: ```suggestion
> [!NOTE]
> Make sure the hardware acceleration is well configured before reading this section.
```

   - For Intel iGPUs, follow the instructions from [intel-compute-runtime](https://github.com/intel/compute-runtime/releases). If you are using the docker image from jellyfin/jellyfin, this step can be skipped.

> [!NOTE]

> > Tone mapping on Intel VAAPI needs an iGPU that support 10-bit decoding.

Reviewer 2: ```suggestion
> Tone mapping on Intel VAAPI needs an iGPU that supports 10-bit decoding.
```
